lib/CamDiag.py

```diff
@@ -588,18 +619,8 @@ def create_plots(self):
                 msg = "Plotting file '{}' is missing. Script is ending here.".format(plot_script_path)
                 end_diag_script(msg)
 
-            #Create regridding script import statement:
-            plot_func_import_statement = "from {} import {}".format(plot_func_name, plot_func_name)
-
-            #Run regridding script import statement:
-            exec(plot_func_import_statement)
-
-            #Create actual function call:
-            plot_func = plot_func_name+\
-            '(case_name, model_rgrid_loc, data_name, data_loc, var_list, data_list, plot_location)'
-
-            #Evaluate (run) plotting script function:
-            eval(plot_func)
+            plot_func_args = [case_name, model_rgrid_loc, data_name, data_loc, var_list, data_list, plot_location]
```

Comment:
It might make sense to add all of the extra `opt` code you have in `create_climo` here as well.  If the code works as expected then it shouldn't modify the current behavior, but will allow one to add arguments to plotting scripts in the future.